### ISSUE TITLE
Documentation wish item

### ISSUE THREAD

**Issue by nateq314** [solidjs/solid#515]
Would love to see a section on the website specifically and directly answering the following for people coming from React and/or Vue:

1) Why another framework?
2) How is this different from React / Vue? What are the relative advantages?

**Comment by ryansolid** [comment]
Yeah I wasn't sure how strong handed I should be.

There is this section: https://www.solidjs.com/docs/1.0.0#comparison-with-other-libraries (looks like the link to hash isn't working properly).

Do you think this blog post is too opinionated to just link [Why SolidJS: Do we need another JS UI Library](https://dev.to/ryansolid/why-solidjs-do-we-need-another-js-ui-library-1mdc). While I have an article for pretty much every topic this is by far my least favorite. I don't like tearing at other libraries with things that fringe on opinion.

I think I probably should address this differently. There is a difference between my motivations for creating a new library and the benefits today.

**Comment by Mihir-Bhasin** [comment]
I would like to work on it can u elaborate more and where I i have to do the changes..

**Comment by tomByrer** [comment]
> I don't like tearing at other libraries with things that fringe on opinion.

You're good at pointing out things you've learned from other frameworks; maybe go with that angle?

**Comment by ryansolid** [comment]
Yeah I think if I do this. I'm going to have to handle this as it is delicate.

**Comment by Salemsa89** [comment]
Aaa

**Comment by Salemsa89** [comment]
Close
